### ISSUE TITLE
Looping through photos in example raises deprecation notice

### ISSUE THREAD

**Issue by daltonrooney** [craftcms/element-api#71]
In your Single Entry Endpoint example, your loop through the photos causes a deprecation notice in Craft 3:
```
// Create an array of all the photo URLs
$photos = [];
foreach ($entry->photos as $photo) {
   $photos[] = $photo->url;
}
```
Message:

> Looping through element queries directly has been deprecated. Use the all() function to fetch the query results before looping over them.

Should we be doing something different?

**Comment by timkelty** [comment]
Yep. `foreach ($entry->photos->all()`, as the message says.

**Comment by daltonrooney** [comment]
👍
Thank you.

**Comment by brandonkelly** [comment]
Sorry, just fixed the code examples in the readme (0a2f8797867045096a6989e2cd9319ab90e1a75f).
